docs/product-release-notes/src/main/asciidoc/rn-rhpam-701-con.adoc

```diff
@@ -0,0 +1,58 @@
+* Business Central and Business Central Monitoring do not apply the correct style to the navigation bar [https://issues.jboss.org/browse/RHPAM-985[RHPAM-985]]
+* The change from server "template" and "container" to "configuration" and "deployment unit" is not consistent across UI [https://issues.jboss.org/browse/RHPAM-955[RHPAM-955]]
+* Newly created persistable data objects are not automatically added to the list of Project Persistable Data Objects [https://issues.jboss.org/browse/RHPAM-938[RHPAM-938]]
+* Milestones: Tooltip with sort order hangs after sorting [https://issues.jboss.org/browse/RHPAM-912[RHPAM-912]]
+* Guided Decision Table: Cannot add a row into table with a work item [https://issues.jboss.org/browse/RHPAM-666[RHPAM-666]]
+* Tasks: First Due Date update writes null to task log [https://issues.jboss.org/browse/RHPAM-609[RHPAM-609]]
+* Switching to Project view redirects to the list of assets [https://issues.jboss.org/browse/RHPAM-556[RHPAM-556]]
+* Error appears after using the Contributors tab [https://issues.jboss.org/browse/RHPAM-535[RHPAM-535]]
+* It is possible to create two project with the same name when using Duplicate [https://issues.jboss.org/browse/RHPAM-533[RHPAM-533]]
+* In Process Designer, open an XML editor instead of a 'sorry' screen when a process cannot be opened [https://issues.jboss.org/browse/RHPAM-447[RHPAM-447]]
+* Process Designer Toolbox icons are cut-off and animations are slow [https://issues.jboss.org/browse/RHPAM-431[RHPAM-431]]
+
+== {KIE_SERVER} 
+
+* KieScanner does not load new rules [https://issues.jboss.org/browse/RHPAM-1124[RHPAM-1124]]
+* Execution Servers: ContainerNotFound error thrown when removing the container [https://issues.jboss.org/browse/RHPAM-899[RHPAM-899]]
+
+== {PRODUCT} on OpenShift
+
+* Broken link to rhpam70-businesscentral-indexing-openshift in image stream file [https://issues.jboss.org/browse/RHPAM-1163[RHPAM-1163]]
+* Maven repo parameter issues in OpenShift templates [https://issues.jboss.org/browse/RHPAM-1166[RHPAM-1166]]
+* Remove the duplicated KIE_SERVER_BYPASS_AUTH_USER required parameter from KIE server template [https://issues.jboss.org/browse/RHPAM-1152[RHPAM-1152]]
+* Drools 7 KieServices.Factory.get() returns null [https://issues.jboss.org/browse/RHPAM-1150[RHPAM-1150]]
+* AMQ readiness check fails on the rhpam70-authoring-ha OpenShift template [https://issues.jboss.org/browse/RHPAM-1150[RHPAM-1150]]
+* Align the value of the RHPAM_TX_ISOLATION environment variable on all templates [https://issues.jboss.org/browse/RHPAM-1140[RHPAM-1140]]
+* Templates remove Smart Router port/protocol configuration [https://issues.jboss.org/browse/RHPAM-1119[RHPAM-1119]]
+* Templates set as non-bindable [https://issues.jboss.org/browse/RHPAM-1117[RHPAM-1117]]
+* Smart Router Servers view error [https://issues.jboss.org/browse/RHPAM-1029[RHPAM-1029]]
+* Case audit: Duplicated entries for Sub-process [https://issues.jboss.org/browse/RHPAM-1029[RHPAM-1029]]
```

Comment:
The same here: the link should be https://issues.jboss.org/browse/RHPAM-877 based on issue title